yeat/workflow/Paired

```diff
@@ -78,3 +78,26 @@ rule penguin:
         samtools index {params.outdir}/unpolished_contigs.sorted.bam
         pilon --genome {params.outdir}/unpolished_contigs.fasta --bam {params.outdir}/unpolished_contigs.sorted.bam --output {params.outdir}/contigs
         """
+
+
+rule velvet:
+    output:
+        contigs="analysis/{sample}/paired/{label}/velvet/contigs.fasta"
+    input:
+        read1="seq/downsample/{sample}/paired/{sample}.R1.fq.gz",
+        read2="seq/downsample/{sample}/paired/{sample}.R2.fq.gz"
+    conda:
+        "velvet"
```

Comment:
I'm going to reiterate my suggestion that this conda environment has a `velvet-` prefix to act as a kind of namespace to reduce the (admittedly already low) likelihood of a collision with an existing environment.